bundle/Form/Type/SearchTagsType.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Netgen\TagsBundle\Form\Type;
+
+use Symfony\Component\Form\Extension\Core\Type\SearchType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+class SearchTagsType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
```

Comment:
No need for this method too:

- method should be specified in the template (`<form method="get"`) and we can leave CSRF protection on, it doesn't interfere with anything.

Reply:
If I leave CSRF protection on, it will be visible in the query string. Why would we want that?  And the HTML search form in https://github.com/netgen/NetgenAdminUIBundle/pull/38 will never be valid.

Reply:
Hm, okay, forgot that this is a `GET` method. Then CSRF protection indeed does not make sense.

Reply:
Removed the form class completely, added a HTML form. Done in https://github.com/netgen/TagsBundle/commit/f4a8ea03a3e319d60e3f799889a2fd0b12598249